pyproject.toml

```diff
@@ -12,8 +12,57 @@ addopts = [
 omit = [".env/*"]
 sort = "Cover"
 
-#[report]
-#sort = Cover
-#omit =
-#    .env/*
-#    backtracking/*
+[tool.codespell]
+ignore-words-list = "3rt,ans,crate,damon,fo,followings,hist,iff,kwanza,mater,secant,som,sur,tim,zar"
+skip = "./.*,*.json,ciphers/prehistoric_men.txt,project_euler/problem_022/p022_names.txt,pyproject.toml,strings/dictionary.txt,strings/words.txt"
+
+[tool.ruff]
+ignore = [      # `ruff rule S101` for a description of that rule
+    "B904",     # B904: Within an `except` clause, raise exceptions with `raise ... from err`
```

Comment:
Um, I didn't mean to have the description of the rule but rather an actual reason as to why are we ignoring it. It could be as simple as "will be fixed in the future" or some XYZ explanation. I'm not aware of these ignores so I can't comment.

Reply:
You are being too picky on this.  If a contributor wants to try removing one of these then they can submit a pull request.  The PR review will be the time to discuss the pros and cons of selecting vs. ignoring each rule.